### DISCUSSION TITLE
Vegdri data

### DISCUSSION THREAD

**Discussion by tmart234** [earthaccess-dev/earthaccess#826] [Q&A]
I see that the vegdri data is available on earth data under vegetation monitoring, but using earth access library, I'm not finding vegdri (but I am finding other data). Also when searching via a URL/curl, I also find it exists.
Any help appreciated 
Thanks

**Comment by chuckwondo** [comment]
Can you provide more details? What exactly did you do on Earthdata to find what you're looking for? What exactly did you try with earthaccess that is not finding the same thing? What exactly did you use with curl?
